ledger/ledger-api-common/src/main/scala/com/digitalasset/ledger/api/Identifiers.scala

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api
+
+import com.daml.ledger._
+import com.daml.ledger.api.v1.transaction.{Transaction, TransactionTree}
+
+/**
+  * Extracts identifiers from Protobuf messages to correlate traces.
+  */
+object Identifiers {
```

Comment:
Could you rename this to `TraceIdentifiers` please?

Reply:
Of course, done.